doc/cygnus-ngsi/installation_and_administration_guide/configuration_examples.md

```diff
@@ -1,47 +1,470 @@
+cygnus-ngsi.sources.src...
+
+# sinks
+cygnus-ngsi.sinks.sink.type = ...
+cygnus-ngsi.sinks.sink.channel = ch
+cygnus-ngsi.sinks.sink...
+
+# channels
+cygnus-ngsi.channels.ch.type = ...
+cygnus-ngsi.channels.ch...
+```
+
+[Top](#top)
+
+### <a name="section1.2"></a>Scenarios involving multiple persistence backends
+Similar as the previous one, in this case there is one channel and one sink per destination backend.
+
+```
++-------------------------------------+
+|              +-----+     +-------+  |
+|        +-----| ch1 |-----| sink1 |  |
+|        |     +-----+     +-------+  |
+|  +-----+     +-----+     +-------+  |
+|  | src |-----| ch2 |-----| sink2 |  |
+|  +-----+     +-----+     +-------+  |
+|        |       ...          ...     |
+|        |     +-----+     +-------+  |
+|        +-----| ch3 |-----| sink3 |  |
+|              +-----+     +-------+  |
++-------------------------------------+
+```
+
+Its agent configuration file is:
+
+```
+$ cat /usr/cygnus/conf/agent_multibackend.conf
+# declarations
+cygnus-ngsi.sources = src
+cygnus-ngsi.sinks = sink1 sink2 ... sinkN
+cygnus-ngsi.channels = ch1 ch2 ... chN
+
+# sources
+cygnus-ngsi.sources.src.type = http
+cygnus-ngsi.sources.src.channels = ch1 ch2 ... chN
+cygnus-ngsi.sources.src.port = 5050
+cygnus-ngsi.sources.src...
+
+# sinks
+cygnus-ngsi.sinks.sink1.type = ...
+cygnus-ngsi.sinks.sink1.channel = ch1
+cygnus-ngsi.sinks.sink1...
+
+cygnus-ngsi.sinks.sink2.type = ...
+cygnus-ngsi.sinks.sink2.channel = ch2
+cygnus-ngsi.sinks.sink2...
+
+...
+
+cygnus-ngsi.sinks.sinkN.type = ...
+cygnus-ngsi.sinks.sinkN.channel = chN
+cygnus-ngsi.sinks.sinkN...
+
+# channels
+cygnus-ngsi.channels.ch1.type = ...
+cygnus-ngsi.channels.ch1...
+
+cygnus-ngsi.channels.ch2.type = ...
+cygnus-ngsi.channels.ch2...
+
+...
+
+cygnus-ngsi.channels.chN.type = ...
+cygnus-ngsi.channels.chN...
+```
+
+Use multi-backend configuration when you simply want the same NGSI context data to be replicated in each one of the backends. This is possible thanks to the default channel selector, which [replicates](https://flume.apache.org/FlumeUserGuide.html#replicating-channel-selector-default) internal NGSI events to all the channels connected to the source. Such a replication is always done independently of the number of FIWARE service paths you own within your FIWARE service, or the number of kind of entities you have.
 
 [Top](#top)
 
-##<a name="section2"></a>Single source, multiple storages
-To be done
+## <a name="section2"></a>Advanced configurations
+### <a name="section2.1"></a>Scenarios involving multiple FIWARE services
+Cygnus NGSI, as many of the FIWARE components, natively supports multitenancy thanks to the **FIWARE service** concept. NGSI notifications consumed by Cygnus contain a Http header carrying such a value, which is used for segmentating context information in the historical backend in the form of per-service dedicated MySQL/MongoDB/STH database, or HDFS user space, or CKAN organization, etc.
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1420/commits/ea511e5c58c65fc69304692013f33e82ae986f1a